---
post_url: /t/tds-official-project1-discrepencies/171141/353
---
There are no evaluation logs for you, I am not sure which evaluation log you are referring to. Your docker image fails to run the required task because your Dockerfile is misconfigured. Did you follow the test environment setup mentioned in this post before posting your query?

![](https://dub1.discourse-cdn.com/flex013/user_avatar/discourse.onlinedegree.iitm.ac.in/carlton/48/56317_2.png)
[Tds-official-Project1-discrepencies](https://discourse.onlinedegree.iitm.ac.in/t/tds-official-project1-discrepencies/171141/316) [Tools in Data Science](/c/courses/tds-kb/34)

> To replicate the test environment:
> Fetch the github repo’s latest commit before 18th feb use below code for that
> import requests
> import pandas
> DEADLINE = pd.Timestamp("2025-02-18", tz="Asia/Kolkata")
> url = f"https://api.github.com/repos/{owner}/{repo}/commits"
> try :
> response = requests.get(url,headers=github\_headers, timeout=60)
> fetch\_commit = None
> if response.status\_code == 200:
> commits = response.json()
> for commit in commits:
> sha = commit["sha"]
> …

Because if you did, you will realise why your evaluation failed.  
You must replicate the test environment and then if you submission works, you have a legitimate appeal. Otherwise we will not consider it. Please replicate the issue using the test environment as detailed in the post link.

Kind regards